TextTest will write diagnostics in /users/geoff/texttesttmp/texttest.lsfgeoff25Nov17:27:56/carmen/matador/AddTest based on file at /users/geoff/work/master/Testing/Automatic/texttest/log4py.conf
Using Application Matador, checkout /carm/proj/texttest/selftest_data
Using Application Picador, checkout /carm/proj/texttest/selftest_data
Starting dynamic TextTest with options : -a cas -g -tp ArtificialUser/flight_back_in_time
Faking start of external progam: 'python /users/geoff/work/master/Testing/TextTest/texttest.py -a cas -g -tp ArtificialUser/flight_back_in_time > /nfs/vm/texttest/central_texttesttmp/texttest.lsfgeoff25Nov17:27:56/carmen/matador/AddTest/TargetApp/static_gui.casgeoff25Nov17:29:54/dynamic_run.log 2> /nfs/vm/texttest/central_texttesttmp/texttest.lsfgeoff25Nov17:27:56/carmen/matador/AddTest/TargetApp/static_gui.casgeoff25Nov17:29:54/dynamic_errors.log'
